### PR TITLE
Align files

### PR DESCRIPTION
**Pull request by architectbot**
This PR is generated by https://github.com/giantswarm/github, it will be merged automatically once all requirements are met

### CODE REVIEW

**Comment by ubergesundheit** [comment]
@giantswarm/team-cabbage please check additional changes by me

**Review by weatherhog — APPROVED** [review]
LGTM

File: helm/velero/templates/_helpers.tpl [hunk 1]
```diff
@@ -110,3 +110,24 @@ Create the volume snapshot location provider
 {{ default .provider .volumeSnapshotLocation.provider }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "labels.common" -}}
+{{ include "labels.selector" . }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
```
Comment (webwurst) [inline helm/velero/templates/_helpers.tpl:119]: Twice the same label
```suggestion
```

**Comment by ubergesundheit** [comment]
Actually, I don't trust myself. I want to add some basic smoke tests

**Comment by webwurst** [comment]
Maybe a test that checks if all created resources have those labels set?
- helm.sh/chart
- app.kubernetes.io/name
- app.kubernetes.io/instance
- app.kubernetes.io/version
- app.kubernetes.io/managed-by
- application.giantswarm.io/team

But just as an idea. Or simpler, get all resources for `app.kubernetes.io/name` and compare count. Feel free to do something else :)

**Pull request merged**